### DISCUSSION TITLE
How can the System Prompt be changed?

### DISCUSSION THREAD

**Discussion by nheidloff** [instructlab/instructlab#1041] [Q&A]
I've added prompt.txt. The warning that the default prompt is used doesn't show up anymore. But I still see "You are an AI language model developed by IBM Research...." in the generated jsonl files. Do I do something wrong?

**Comment by andretost** [comment]
You could also serve the new model via ollama and then use Open-webui with a Modelfile to add your own system prompt, right? I have successfully done that, even though have to admit that my system prompts don't seem to work very well. :-)
